Assets/Scripts/Nodes/IsDestinationWithinLimits.cs

```diff
@@ -5,16 +5,16 @@
 
 public class IsDestinationWithinLimits : LeafNode
 {
-	Vector3 _destination;
+	private Hashtable _data;
 
 	public override void InitSelf( Hashtable data )
 	{
-		_destination = (Vector3)data["destination"];
+		_data = data;
 	}
 
 	public override NodeStatus TickSelf()
 	{
-		if ( MathUtils.IsWithinInfiniteVerticalCylinders( _destination, LimitsManager.colliders ) )
+		if ( MathUtils.IsWithinInfiniteVerticalCylinders( (Vector3) _data["destination"], LimitsManager.colliders ) )
```

Comment:
The destination value was being set in the tick method of another node. So when they were run in sequence it would go Init, Init, Tick, Tick and this node would try to read it in the Init before it was set.